ci/jenkins/jobs/projects.yaml

```diff
@@ -98,293 +98,482 @@
+                        variable: CODECOV_TOKEN
+            - timeout:
+                  fail: true
+                  timeout: 120
+                  type: absolute
+            - credentials-binding:
+                  - text:
+                        credential-id: VMC_NETWORK_0
+                        variable: VMC_NETWORK_0
+                  - text:
+                        credential-id: VMC_NETWORK_1
+                        variable: VMC_NETWORK_1
+                  - text:
+                        credential-id: VCENTERNAME
+                        variable: VCENTERNAME
+                  - text:
+                        credential-id: DATACENTERNAME
+                        variable: DATACENTERNAME
+                  - text:
+                        credential-id: RESOURCEPOOLPATH
+                        variable: RESOURCEPOOLPATH
+                  - text:
+                        credential-id: VC_USER
+                        variable: VC_USER
+                  - text:
+                        credential-id: GOVC_PASSWORD
+                        variable: GOVC_PASSWORD
+                  - text:
+                        credential-id: VC_HOST
+                        variable: VC_HOST
+                  - text:
+                        credential-id: DATA_STORE
+                        variable: DATA_STORE
+                  - text:
+                        credential-id: VC_CLUSTER
+                        variable: VC_CLUSTER
+                  - text:
+                        credential-id: VIRTUAL_MACHINE
+                        variable: VIRTUAL_MACHINE
+                  - text:
+                        credential-id: DOCKER_USERNAME
+                        variable: DOCKER_USERNAME
+                  - text:
+                        credential-id: DOCKER_PASSWORD
+                        variable: DOCKER_PASSWORD
+                  - text:
+                        credential-id: AZURE_APP_ID
+                        variable: AZURE_APP_ID
+                  - text:
+                        credential-id: AZURE_PASSWORD
+                        variable: AZURE_PASSWORD
+                  - text:
+                        credential-id: AZURE_SUBSCRIPTION_ID
+                        variable: AZURE_SUBSCRIPTION_ID
+                  - text:
+                        credential-id: AZURE_TENANT_ID
+                        variable: AZURE_TENANT_ID
+          publishers:
+              - archive:
+                    allow-empty: true
+                    artifacts: "logs.tar.gz"
+                    case-sensitive: true
+                    default-excludes: true
+                    fingerprint: false
+                    only-if-success: false
+      - '{test_name}-for-pull-request':
+          test_name: nephe-test-e2e-aws-upgrade
+          node: 'nephe'
+          description: 'This is the {test_name}.'
+          branches:
+            - ${{sha1}}
```

Comment:
Is this still a pull request job?

Reply:
yes. 2 news jobs are being added with PR for upgrade.